-- Logs begin at Wed 2022-11-02 18:57:16 IST, end at Wed 2022-11-02 18:57:21 IST. --
Nov 02 18:57:17 datacore-PowerEdge-R740 sudo[3265150]: pam_unix(sudo:session): session closed for user root
Nov 02 18:57:17 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix stack-volumes-lvmdriver-1" returned: 0 in 0.729s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:17 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix stack-volumes-lvmdriver-1 {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:17 datacore-PowerEdge-R740 sudo[3265206]:    stack : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/local/bin/cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix stack-volumes-lvmdriver-1
Nov 02 18:57:17 datacore-PowerEdge-R740 sudo[3265206]: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 02 18:57:18 datacore-PowerEdge-R740 sudo[3265206]: pam_unix(sudo:session): session closed for user root
Nov 02 18:57:18 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix stack-volumes-lvmdriver-1" returned: 0 in 0.721s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:18 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1 {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:18 datacore-PowerEdge-R740 sudo[3265245]:    stack : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/local/bin/cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1
Nov 02 18:57:18 datacore-PowerEdge-R740 sudo[3265245]: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 02 18:57:18 datacore-PowerEdge-R740 sudo[3265245]: pam_unix(sudo:session): session closed for user root
Nov 02 18:57:18 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1" returned: 0 in 0.631s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:18 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:18 datacore-PowerEdge-R740 sudo[3265253]:    stack : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/local/bin/cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool
Nov 02 18:57:18 datacore-PowerEdge-R740 sudo[3265253]: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 02 18:57:19 datacore-PowerEdge-R740 sudo[3265253]: pam_unix(sudo:session): session closed for user root
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool" returned: 0 in 0.635s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): env LC_ALL=C lvm version {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "env LC_ALL=C lvm version" returned: 0 in 0.015s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:19 datacore-PowerEdge-R740 sudo[3265431]:    stack : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/local/bin/cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool
Nov 02 18:57:19 datacore-PowerEdge-R740 sudo[3265431]: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 02 18:57:19 datacore-PowerEdge-R740 sudo[3265431]: pam_unix(sudo:session): session closed for user root
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool" returned: 0 in 0.632s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: INFO cinder.volume.drivers.lvm [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Enabling LVM thin provisioning by default because a thin pool exists.
Nov 02 18:57:19 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): env LC_ALL=C lvm version {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:20 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "env LC_ALL=C lvm version" returned: 0 in 0.015s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:20 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:20 datacore-PowerEdge-R740 sudo[3265437]:    stack : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/local/bin/cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool
Nov 02 18:57:20 datacore-PowerEdge-R740 sudo[3265437]: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 02 18:57:20 datacore-PowerEdge-R740 sudo[3265437]: pam_unix(sudo:session): session closed for user root
Nov 02 18:57:20 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix --readonly stack-volumes-lvmdriver-1/stack-volumes-lvmdriver-1-pool" returned: 0 in 0.673s {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:422}}
Nov 02 18:57:20 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG cinder.volume.drivers.lvm [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Updating volume stats {{(pid=3264900) _update_volume_stats /opt/stack/cinder/cinder/volume/drivers/lvm.py:213}}
Nov 02 18:57:20 datacore-PowerEdge-R740 cinder-volume[3264900]: DEBUG oslo_concurrency.processutils [None req-8f1ec79a-b16e-4247-9910-a7f0fe5e918c None None] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix stack-volumes-lvmdriver-1 {{(pid=3264900) execute /usr/local/lib/python3.8/dist-packages/oslo_concurrency/processutils.py:384}}
Nov 02 18:57:20 datacore-PowerEdge-R740 sudo[3265447]:    stack : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/local/bin/cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix stack-volumes-lvmdriver-1
Nov 02 18:57:20 datacore-PowerEdge-R740 sudo[3265447]: pam_unix(sudo:session): session opened for user root by (uid=0)
